tests/test_vocabulary_handler.py

```diff
@@ -0,0 +1,275 @@
+from lxml import objectify, etree
+import pytest
+
+from controlled_vocabularies.vocabulary_handler import VocabularyHandler
+from . import factories
+
+pytestmark = pytest.mark.django_db
+
+# Namespaces
+NS = {
+    'rdf': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#',
+    'dc': 'http://purl.org/dc/elements/1.1/',
+    'rdfs': 'http://www.w3.org/2000/01/rdf-schema#',
+    'purl': 'http://purl.org/NET/UNTL/vocabularies/formats/'
+}
+
+
+def test_xml_response():
+    vocab = factories.VocabularyFactory()
+    vocab_handler = VocabularyHandler().xml_response(vocab)
+    assert vocab_handler.vocab == vocab
+    assert isinstance(vocab_handler, VocabularyHandler)
+
+
+@pytest.fixture
+def vocab_file():
```

Comment:
Can we give this a more descriptive name? Something to let us know that it will return the xml format?